### PR TITLE
Ensure all forms have a csrf token input

### PR DESCRIPTION
**Pull request by afiore**
Also add a couple of style tweaks

### CODE REVIEW

File: app/views/shipping/index.scala.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
-@(shipping: model.ShippingDetails)
+@(shipping: model.ShippingDetails)(implicit rh: RequestHeader)
 
-@main(s"${shipping.name.split("\\s").map(_.capitalize).mkString(" ")} | Choose a Package | Subscriptions | The Guardian", footer = fragments.footer.intl_faq()) {
+@capitalizedName=@{shipping.name.split("\\s").map(_.capitalize).mkString(" ")}
```
Comment (jennysivapalan) [inline app/views/shipping/index.scala.html]: I think `@{shipping.name.split("\\s").map(_.capitalize).mkString(" ")}` should be in `ShippingDetails`

Comment (afiore) [inline app/views/shipping/index.scala.html]: Why not...

**Comment by TesterSpike** [comment]
I can confirm that on /digital/country selecting 'continue' will now successfully take you to /checkout

**Comment by jennysivapalan** [comment]
:+1:

**Comment by afiore** [comment]
Verified on prod

**Pull request merged**